src/usePersistedState.js

```diff
@@ -27,19 +27,32 @@ const usePersistedState = (initialState, key, { get, set }) => {
     };
   }, []);
 
+  const persistValue = useCallback(newState => {
+    // persist to localStorage
+    set(key, newState);
+
+    // inform all of the other instances in this tab
+    globalState.current.emit(newState);
+  }, [key, set]);
+
   // Only persist to storage if state changes.
   useEffect(
```

Comment:
thinking about it some more, it may no longer need this bit since the only way the state will have changed is if a new value has been persisted, so no need to persist it again

Reply:
I have removed it now